src/pages/Landing.tsx

```diff
@@ -0,0 +1,13 @@
+import Footer from '../components/molecules/Footer';
```

Comment:
rename this component to basePage and move it to src/components/templates, please

Reply:
Done +1